test/shared/src/main/scala/zio/test/Spec.scala

```diff
@@ -22,8 +22,8 @@ import Spec._
 
 /**
  * A `Spec[L, T]` is the backbone of _ZIO Test_. Every spec is either a suite,
- * which contains other specs, or a test of type `T`. All specs are annotated
- * with labels of type `L`.
+ * which contains other specs, or a test of type `T` or an already executed test with test result. 
```

Comment:
Not sure if this is the right wording but as per https://github.com/zio/zio/blob/master/test/shared/src/main/scala/zio/test/package.scala#L93

Spec is used for already run tests.

Reply:
A `Spec` is just a collection of tests of some type `T`. Depending on what the type is those tests could be executed or not. As a user when you are interacting with Specs they have generally not been executed yet but are a description of the tests (see for example `ZSpec`). That's why you can do all these nice operations on them like ignore them, retry them, provide them with environments, etc... An `ExecutedSpec` is one particular kind of Spec that describes a test that has been run. ZIO Test works with `ExecutedSpec` for reporting results and if want to modify ZIO Test for advanced use case like reporting results in a different way you might use it, but for most users the specs they are creating in test suites will not have been executed yet.

Reply:
I'll revert this change.